lib/cache/PackFileCacheStrategy.js

```diff
@@ -6,14 +6,53 @@
 "use strict";
 
 const FileSystemInfo = require("../FileSystemInfo");
-const LazySet = require("../util/LazySet");
+const PathLazySet = require("../util/PathLazySet");
+const PathSet = require("../util/PathSet");
+const { absoluteToRequest, requestToAbsolute } = require("../util/identifier");
 const makeSerializable = require("../util/makeSerializable");
 const memorize = require("../util/memorize");
 const { createFileSerializer } = require("../util/serialization");
 
 /** @typedef {import("../Cache").Etag} Etag */
 /** @typedef {import("../logging/Logger").Logger} Logger */
+/** @template T @typedef {import("../util/LazySet")<T>} LazySet<T> */
 /** @typedef {import("../util/fs").IntermediateFileSystem} IntermediateFileSystem */
+/** @typedef {import("./types").CacheStrategyOptions} CacheStrategyOptions */
+/** @typedef {import("./types").CacheWriteReadAbsolutePath} CacheWriteReadAbsolutePath */
+
+const ABSOLUTE_PATH_IDENTIFIER_REGEX = /\/(\/.*)$/;
+const RESOLVE_IDENTIFIER_REGEX = /path=(.*)\/request=.*$/;
+
+/**
+ * @param {string} cacheLocation cache location
+ * @param {boolean} contextify contextify flag
+ * @returns {CacheWriteReadAbsolutePath} context
+ */
+function createAbsolutePathsWriteReadMethods(cacheLocation, contextify) {
```

Comment:
should be moved into seralization.js

Reply:
Should I export it. For instance I wont to write my `<blabla>Serializer`?